website/docs/r/service_v1.html.markdown

```diff
@@ -383,6 +383,8 @@ compressed. Default `0`.
 * `response_condition` - (Optional) Name of already defined `condition` to apply. This `condition` must be of type `RESPONSE`. For detailed information about Conditionals,
 see [Fastly's Documentation on Conditionals][fastly-conditionals].
 * `placement` - (Optional) Where in the generated VCL the logging call should be placed; one of: `none` or `waf_debug`.
+* `server_side_encryption` - (Optional) Specify what type of server side encryption should be used. Can be either `AES256` or `aws:kms`.
```

Comment:
Apologies, @alpagot, these changes need to be in the `website_src` directory which contains the templates and not the `website` directory which contains the rendered output. I.e. in this file: https://github.com/fastly/terraform-provider-fastly/blob/master/website_src/docs/r/service_v1.html.markdown.tmpl

You'll then need to run `make website` to generate the docs before committing both files back.

Reply:
Ah my apologies!